### PR TITLE
some additional embed base styling

### PR DESCRIPTION
**Pull request by vellip**

### CODE REVIEW

File: apps/embed/templates/meinberlin_embed/embed.html [hunk 1]
```diff
@@ -20,10 +20,10 @@
 <body data-url="{% url 'project-detail' slug %}">
     <header class="embed-header">
         {% if request.user.is_anonymous %}
-            <a href="{% url 'account_login' %}">{% trans 'Login' %}</a>
+            <a href="{% url 'account_login' %}" class="js-embed-login button button--light">{% trans 'Login' %}</a>
```
Comment (xi) [inline apps/embed/templates/meinberlin_embed/embed.html]: I think the `js-*` class sneaked back in in the merge

Comment (vellip) [inline apps/embed/templates/meinberlin_embed/embed.html]: ah i was afraid something like that would happen.

Comment (vellip) [inline apps/embed/templates/meinberlin_embed/embed.html]: Now it should be the same as it is in master.

**Comment by xi** [comment]
Much better

**Pull request merged**